test/unit/presenters/publishing_api/publication_presenter_test.rb

```diff
@@ -39,7 +39,7 @@ def present(edition)
           browse_pages: [],
           topics: [],
         },
-        documents: ["<section class=\"attachment embedded\" id=\"attachment_#{publication.attachments.first.id}\">\n  <div class=\"attachment-thumb\">\n      <a aria-hidden=\"true\" class=\"thumbnail\" tabindex=\"-1\" href=\"/government/publications/publication-title/#{publication.attachments.first.content_id}\"><img alt=\"\" src=\"https://www.test.gov.uk/assets/whitehall/pub-cover-html.png\" /></a>\n  </div>\n  <div class=\"attachment-details\">\n    <h3 class=\"title govuk-!-font-size-27 govuk-!-font-weight-regular\"><a href=\"/government/publications/publication-title/#{publication.attachments.first.content_id}\">#{publication.attachments.first.title}</a></h3>\n    <p class=\"metadata\">\n        <span class=\"type\">HTML</span>\n    </p>\n\n\n  </div>\n</section>"],
+        documents: ["<section class=\"attachment embedded\" id=\"attachment_#{publication.attachments.first.id}\">\n  <div class=\"attachment-thumb\">\n      <a aria-hidden=\"true\" class=\"thumbnail\" tabindex=\"-1\" href=\"/government/publications/publication-title/#{publication.attachments.first.title}\"><img alt=\"\" src=\"https://www.test.gov.uk/assets/whitehall/pub-cover-html.png\" /></a>\n  </div>\n  <div class=\"attachment-details\">\n    <h3 class=\"title govuk-!-font-size-27 govuk-!-font-weight-regular\"><a href=\"/government/publications/publication-title/#{publication.attachments.first.title}\">#{publication.attachments.first.title}</a></h3>\n    <p class=\"metadata\">\n        <span class=\"type\">HTML</span>\n    </p>\n\n\n  </div>\n</section>"],
```

Comment:
As this change reverses some of the changes you made in the previous commit, I'd be tempted to squash these middle two commits together. I.e. make one super commit of "use content IDs for non-English HTML attachments".